order/buyer/issue/indicators.md

```diff
@@ -92,34 +92,24 @@ The order or line having logistics status `Open`, `Produced`, `ReadyToShip` or `
 
 {% page-ref page="propose-when-accepted.md" %}
 
-## Single delivery
+## Single delivery order line behavior
```

Comment:
Suggest to add a hint or some other note to contact Support@tradecloud when a buyer wants to use the indicator (or deliveries) endpoint. 
As sending the primary order line is not really "make it hard for use and easy for the buyer". 
For the delivery endpoint we have already this ticket on the backlog: 
endpoint

Reply:
added to this and cancel, complete and receive-goods pages